blog-redirects.properties

```
@@ -27,3 +27,4 @@
 /blog,/microprofile/2021/03/31/microprofile-config-2.0.html=/blog/2021/03/31/microprofile-config-2.0.html
```

```suggestion
/blog/microprofile/2021/03/31/microprofile-config-2.0.html=/blog/2021/03/31/microprofile-config-2.0.html
```
This `,` looks like a bug/typo

---

it is not; that is partially what the redirect is there for